addons/cni-cilium/cilium.yaml

```diff
@@ -70,8 +70,6 @@ data:
   cilium-endpoint-gc-interval: "5m0s"
   nodes-gc-interval: "5m0s"
   skip-cnp-status-startup-clean: "false"
-  # Disable the usage of CiliumEndpoint CRD
-  disable-endpoint-crd: "false"
```

Comment:
I'm not sure about this change, I have to double-check where is it coming from

Reply:
https://github.com/cilium/cilium/commit/c6bc4d61c2d807decdb61674c5115ad555580df4